src/Result/index.tsx

```diff
@@ -1,5 +1,19 @@
 import { Result as AntdResult, ResultProps as AntdResultProps } from 'antd';
+import React from 'react';
 
 export const Result: typeof AntdResult = AntdResult;
 
 export type ResultProps = AntdResultProps;
+
+export function EntityNotFound(props: { entity: string; id: string }) {
+  return (
+    <Result
+      status={'404'}
+      title={
+        <>
+          {props.entity} #{props.id} konnte nicht gefunden werden.
```

Comment:
```suggestion
          {props.entity} {props.id} konnte nicht gefunden werden.
```

Makes copy-paste easier and reduces the chance of error, especially when dealing with non-numeric identifiers.

Reply:
I do not like  "Kontrolle 12 konnte nicht gefunden werden." very much. What do you think about "Kontrolle mit ID:12 konnte nicht gefunden werden."

Reply:
Looks good to me. Extra symbols or text just seem unnecessarily verbose and noisy to me.

Let's see how much we can take away without losing out on information:

```
Institut mit ID:4a710b37-e8e2-4a80-982f-b4cdd67a04f1 konnte nicht gefunden werden.
Institut #4a710b37-e8e2-4a80-982f-b4cdd67a04f1 konnte nicht gefunden werden.
Institut 4a710b37-e8e2-4a80-982f-b4cdd67a04f1 konnte nicht gefunden werden.
Institut 4a710b37-e8e2-4a80-982f-b4cdd67a04f1 konnte nicht gefunden werden
Institut 4a710b37-e8e2-4a80-982f-b4cdd67a04f1 nicht gefunden
```

Reply:
Apparently, you have a very strong opinion here. I would still prefer the addition "with ID:" because it just sounds better to me: `Institut mit ID:4a710b37-e8e2-4a80-982f-b4cdd67a04f1 nicht gefunden`.
But since this is simply a matter of linguistic feeling and taste, I guess there is no point in discussing it....

Reply:
> Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.

Antoine de Saint-Exupery